parser-typechecker/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -926,19 +926,26 @@ diffNamespace :: InputPattern
 diffNamespace = InputPattern
   "diff.namespace"
   []
-  [(Required, pathArg), (Required, pathArg)]
+  [(Required, pathArg), (Optional, pathArg)]
   (P.column2
     [ ( "`diff.namespace before after`"
       , P.wrap
         "shows how the namespace `after` differs from the namespace `before`"
       )
+    , ( "`diff.namespace before`"
+      , P.wrap
+        "shows how the namespace `before` differs from the current namespace"
```

Comment:
for symmetry with the previous example, let's say "shows how the current namespace differs from the namespace `before`"

Reply:
You're right, that makes more sense